site/en/guides/border-router/espressif-esp32.md

```diff
@@ -75,27 +79,12 @@ Done
 * IPv6 bidirectional connectivity.
 * SRP service registration and advertising proxy.
 * mDNS discovery proxy.
+* NAT64
+* Multicast forwarding
+* Web GUI based REST API.
+* OTA
```

Comment:
The list formatting should be consistent, either all should end in periods, or all don't end in periods.